tools/ack-discover/controller.py

```diff
@@ -133,3 +157,70 @@ def get_runtime_and_aws_sdk_version(writer, repo, image_version):
         # proper Git tags for releases...
         pass
     return runtime_version, aws_sdk_version
+
+
+def get_github_issue(writer, gh, service):
```

Comment:
nit: Perhaps a more specific name?
```suggestion
def get_controller_request_issue(writer, gh, service):
```

Reply:
done.